docs/build/guides/conversions/address-conversions.mdx

```diff
@@ -6,50 +6,35 @@ description: Convert an address to other types
 
 import { CodeExample } from "@site/src/components/CodeExample";
 
-The `Address` is an opaque type that could represent an "account" on the Stellar network (i.e., a keypair), or a "contract." From Soroban's point of view, it doesn't really matter which it is. The "account" variety of these addresses are typically displayed as a `G...` public address, and the "contract" variety is typically displayed as a `C...` address. An address may also be displayed in other formats such as a 32 byte array, a string, or an ScVal type. The Soroban SDKs provide methods that easily convert an address to any of these types.
+The `Address` is an opaque type that represents either a 'default' externally owned account on the Stellar network, or a contract (that may also provide logic for custom externally owned accounts, see [authorization docs](../../../learn/encyclopedia/security/authorization.mdx#account-abstraction) for details). For the smart contracts it normally doesn't matter which kind of `Address` is used. However, in some contexts it's useful to convert `Address` to/from different data types, such as string or XDR. The conversions have distinctly different purpose depending on whether they happen in the smart contract itself, or in the client code.
 
-## Address to bytesN
+# String conversions
 
-Bytes are a more compact and efficient way to store data in terms of storage optimization and data transmission. In situations where you need to store or transmit an address in a fixed-size, such as for cryptographic operations or data serialization, you need to convert the address to a bytesN format
+The default string format for `Address` is so called 'string key' (or 'strkey'), a short base64-encoded string. For the default externally owned accounts strkey always starts with `G` and for all the contracts it starts with `C`. The exact details on the format can be found [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0023.md), but they are not necessary to understand this section.
```

Comment:
`strkey`s don't involve base64

```suggestion
The default string format for `Address` is a so called 'string key' (or 'strkey'), defined fully in [SEP-23](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0023.md). For the default externally owned accounts strkey always starts with `G` and for all the contracts it starts with `C`.
```

Reply:
Yeah, it's base32 actually, but I guess it's not very relevant.